include/seahorn/Support/SeaLog.hh

```diff
@@ -16,11 +16,13 @@
   ::seahorn::warn_ostream(llvm::errs())                                        \
       .prefix("Info: ")                                                        \
       .suffix(FLOC(__FILE__, __LINE__))
+#undef INFO // workaround(test): to avoid name clash with INFO in doctest.h
```

Comment:
I guess this is acceptable, but I would much rather have a simpler fix.

1. no changes in doctest or seahorn files
2. new header 'sea_doctest.h' that
        1. undefs INFO, WARN, ERR
        2. includes doctest
 3. in all tests, include doctest last

Reply:
done